packages/shared/src/components/widgets/WidgetCard.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { PropsWithChildren, ReactElement } from 'react';
+import classNames from 'classnames';
+
+interface WidgetCardProps extends PropsWithChildren {
+  heading?: string;
+  className?: string;
+}
+
+export const WidgetCard = (props: WidgetCardProps): ReactElement => {
```

Comment:
Nice, didn't we have such a component already for what renders on other pages with sidebar?

Reply:
It is weird but didn't having one, right now for example `SimilarPosts` and `BestDiscussions` have 2 different implementation (I discovered recently working on another ticket) 👀

Reply:
Ah ok, all good I like this was surprised to learn we didn't have it 😂